lib/opFns.js

```diff
@@ -799,6 +799,7 @@ function trap (err) {
 /**
  * Subtracts the amount needed for memory usage from `runState.gasLeft`
  * @method subMemUsage
+ * @param runState
```

Comment:
We normally have the convention to just use ``{Object}`` as a type if this is the most we can get 😄 (see other documentation in the library). If this is not wrong per se (I am not a super-comment-syntax expert) I would like to keep here with this as well.

Reply:
Yup, I'll add the type in